src/utils/form-actions.ts

```diff
@@ -15,13 +16,16 @@ export async function saveSession(session: Session) {
   return data;
 }
 
-export async function getSession(sessionId: string): Promise<Session> {
+export async function getSession(
+  sessionId: string
+): Promise<Session | undefined> {
   const { data } = await getAmplifyBackendClient().models.SessionStorage.get({
     id: sessionId,
   });
 
   if (!data) {
-    throw new Error('Could not retrieve session');
+    logger.warn(`Failed to retrieve session for ID ${sessionId}`);
```

Comment:
where can these logs be seen in the deployed app?

Reply:
Good point, there is a link to the log stream but it does not appear to exist.  I'll ticket that separately.  The logging at least appears when running locally.

Reply:
Ticket: https://nhsd-jira.digital.nhs.uk/browse/CCM-6260